fusor-ember-cli/app/serializers/foreman-task.js

```diff
@@ -1,6 +1,7 @@
 import DS from 'ember-data';
 
-export default DS.RESTSerializer.extend({
+export default DS.ActiveModelSerializer.extend({
+  isNewSerializerAPI: true,
```

Comment:
@isratrade I looked up the reason for isNewSerializerAPI: true:

http://emberjs.com/blog/2015/06/18/ember-data-1-13-released.html#toc_upgrade-guide

If you have customized your serializer, you should upgrade to Ember Data 1.13, check the upgrade guide to see if you need to make any changes, and then set a temporary flag on your Serializer: isNewSerializerAPI. This will opt you into the new serializer API. Once you are on the Ember Data 2.0 train, new Serializer API is the default one, and there is no need for a flag.